Dockerfile.php.busybox

```diff
@@ -0,0 +1,163 @@
+		export GNUPGHOME="$(mktemp -d)"; \
+		gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$GPG_KEY1"; \
+		gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$GPG_KEY2"; \
+		gpg --batch --verify php.tar.xz.asc php.tar.xz; \
+		rm -rf "$GNUPGHOME"; \
+	fi; \
+	tar Jxvf php.tar.xz; \
+	rm php.tar.xz.asc php.tar.xz; \
+	\
+	cd /usr/src/php-$PHP_VERSION/; \
+	ADDITIONAL_OPTIONS=''; \
+	if [ $VERSION = "5.6" ]; then ADDITIONAL_OPTIONS="--with-mysql"; fi; \
+	./configure --build="x86_64-linux-gnu" --enable-fpm --enable-static --disable-shared \
+	--with-fpm-user=www-data --with-fpm-group=www-data --with-config-file-path="$PHP_INI_DIR" \
+	--with-config-file-scan-dir="$PHP_INI_DIR/conf.d" --with-bz2 --enable-bcmath \
+	--enable-calendar --enable-dba --enable-exif --enable-ftp --with-gd --with-gettext \
+	--with-kerberos --with-openssl --with-imap=/usr/lib --with-imap-ssl --with-png-dir=/usr \
+	--with-jpeg-dir=/usr --enable-intl --with-ldap --with-mcrypt --with-mysqli --with-pdo-mysql \
+	--enable-shmop --enable-soap --enable-sockets --enable-sysvmsg --enable-sysvsem \
+	--enable-sysvshm --enable-wddx --with-xmlrpc --with-xsl --enable-zip --enable-mbstring \
+	--enable-mysqlnd --with-curl --with-libedit --with-openssl --with-zlib --with-pcre-regex=/usr \
+	--with-libdir="lib/x86_64-linux-gnu" $ADDITIONAL_OPTIONS; \
+	make -j "$(nproc)"; \
+	make install; \
+
+	# Build and install pecl modules
+	if [ $VERSION = "5.6" ]; then \
+		PECL_MODULES=${PHP5_PECL_MODULES}; \
+	else \
+		PECL_MODULES=${PHP7_PECL_MODULES}; \
+	fi; \
+	for ext in $PECL_MODULES; do echo "\n" | pecl install $ext; done; \
+
+	# Download the ioncube loader extension
+	curl --connect-timeout 10 -o ioncube.tar.gz -fSL "https://downloads.ioncube.com/loader_downloads/ioncube_loaders_lin_x86-64.tar.gz"; \
+	tar -zxvf ioncube.tar.gz; \
+	cp ioncube/ioncube_loader_lin_$VERSION.so $PHP_EXT_PATH/ioncube.so; \
+	rm -Rf ioncube*; \
+
+	# Download the New Relic extension
+	NR_VERSION="$( \
+		curl --connect-timeout 10 -sS https://download.newrelic.com/php_agent/release/ \
+			| sed -n 's/.*>\(.*linux\).tar.gz<.*/\1/p' \
+	)"; \
+	curl --connect-timeout 10 -o nr.tar.gz -fSL "https://download.newrelic.com/php_agent/release/$NR_VERSION.tar.gz"; \
+	tar -xf nr.tar.gz; \
+	cp $NR_VERSION/agent/x64/newrelic-$PHP_EXT_SUFFIX.so $PHP_EXT_PATH/newrelic.so; \
+	rm -rf newrelic-php5* nr.tar.gz; \
+
+	# Gather required runtime files from installed locations and copy them
+	# to the .deploy folder which will serve as the new root folder for the 
+	# final image.
+	CONF_DIR=usr/local/etc/php/conf.d; \
+	\
+	mkdir -p .deploy; \
+	cd .deploy; \
+	# Make the directories we will need at runtime
+	mkdir -p usr/local/bin usr/local/sbin usr/lib usr/local/include/php/main ./$PHP_EXT_PATH $CONF_DIR; \
+	mkdir -p var/lib/php/sessions var/lib/php/opcache run/php; \
+	cp /usr/local/sbin/php-fpm usr/local/sbin/.; \
+	cp /usr/local/include/php/main/php_version.h usr/local/include/php/main/.; \
```

Comment:
Since we have the short and long version of the PHP version in hand, we might consider a simple file with just the version number if you don't want to rely on the ENV downstream.